specifications/api/swagger.yaml

```diff
@@ -1967,26 +1968,32 @@ paths:
   /wallets/{walletId}/migrations:
     get:
       operationId: getShelleyWalletMigrationInfo
-      tags: ["Shelley Migrations"]
+      tags: ["Migrations"]
       summary: Calculate Cost
       description: |
-        <p align="right">status: <strong>stable</strong></p>
+        <p align="right">status: <strong>in development</strong></p>
 
-        Calculate the exact cost of migrating from a Shelley wallet to
-        the specified set of addresses.
+        Calculate the exact cost of sending all funds from particular Shelley wallet
+        to a set of addresses.
       parameters:
         - <<: *parametersWalletId
           name: walletId
       responses: *responsesGetWalletMigrationInfo
 
     post:
       operationId: migrateShelleyWallet
-      tags: ["Shelley Migrations"]
+      tags: ["Migrations"]
       summary: Migrate
       description: |
-        <p align="right">status: <strong>stable</strong></p>
+        <p align="right">status: <strong>in development</strong></p>
 
         Migrate all funds from a Shelley wallet to a specified set of addresses.
+        Create one or several transactions sending all funds from a Shelley wallet to set of addresses.
+        This operation tries to preserve as much as possible the current UTxO shape of a wallet (hence
+        why several transactions may be needed).
```

Comment:
I wonder whether word `why` is needed here :thinking:  @rvl / @jonathanknowles  could you voice your opinion?

Reply:
How about this

> Submit one or more transactions which transfers all funds from a Shelley wallet to a set of addresses.
> This operation attempts to preserve the UTxO "shape" of a wallet as far as possible. That is, coins will not be agglomerated.
> Therefore, if the wallet has a large UTxO set, several transactions may be needed.

Reply:
Reworded in 508e41b4d94be8861ab9c8e407a1584975ad6dfa.